### ISSUE TITLE
simulataneous runs of fskx-models using OpenBUGS 

### ISSUE THREAD

**Issue by Thatsreallycool** [RakipInitiative/ModelRepository#155]
Not an issue with FSK-Lab, but an issue of the communication between R and openBUGS in connection with their communication with FSK-Lab.

If one runs a working fskx-model with openBUGS, the connection to openBUGS is blocked for other users. If two users each run a model using openBUGS (more likely on the server, but also happens on local installation if 2 models are run in close sequence of each other), the user who started the model run first will succeed, the user who started a bit later will get an error message, that the model was implemented incorrectly.

**Comment by schuelet** [comment]
@mfilter this is an edge case and I don't really have an answer, other than providing a more helpful error message. Maybe there is a way in R to tell the openbugs connection to wait but this should probably do the modeller. If FSK-Lab now starts to 'wait' for certain conditions, we might get more problems down the line with KNIME and other services.

**Comment by mfilter** [comment]
Agree, this is a specific scenario (but it might be true for other models as well). I would keep this ticket open. A meaningful error message would for sure be of help.
